### PR TITLE
fix link

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

File: docs/swarm_at_scale/deploy-app.md [hunk 1]
```diff
@@ -276,9 +276,9 @@ the containers at once. This extra credit
 
 2. Try to create Compose file on your own by reviewing the tasks in this tutorial.
 
-    <a href="http://192.168.99.100:8000/compose/compose-file/#entrypoint" target="_blank">The
-    version 2 Compose file format</a> is the best to use. Translate each `docker
-    run` command into a service in the `docker-compose.yml` file. For example,
+    [The version 2 Compose file format](/compose/compose-file.md)
```
Comment (thaJeztah) [inline docs/swarm_at_scale/deploy-app.md]: Do we want to drop the target=blank here? (I hate the HTML markup, but I can understand the thought behind having a target blank)

Comment (SvenDowideit) [inline docs/swarm_at_scale/deploy-app.md]: I may not be the right person to ask - my browser blocks all popups - @londoncalling ?

tbh, https://news.ycombinator.com/item?id=11631292 makes me think we should just forbid it.

Comment (thaJeztah) [inline docs/swarm_at_scale/deploy-app.md]: Yeah, wasn't the biggest fan initially, but thought I'd bring it up :innocent:

**Comment by SvenDowideit** [comment]
@sfsmithcha

**Comment by sfsmithcha** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM

**Pull request merged**